recipes/rank_filter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "rank_filter" %}
+{% set version = "0.4.11" %}
+{% set checksum = "246f5c73ebd03314d9b2cff3ede74330" %}
```

Comment:
I think this might be a little overkill :wink: - we'll end up jinja-ing out the whole thing :smile: 

Would you mind if this were just version, and the other two were inlined?